content/en/blog/_posts/2015-10-00-Some-Things-You-Didnt-Know-About-Kubectl_28.md

```diff
@@ -4,7 +4,7 @@ date: 2015-10-28
 slug: some-things-you-didnt-know-about-kubectl_28
 url: /blog/2015/10/Some-Things-You-Didnt-Know-About-Kubectl_28
 ---
-[kubectl](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/kubectl-overview.md) is the command line tool for interacting with Kubernetes clusters. Many people use it every day to deploy their container workloads into production clusters. But there’s more to kubectl than just `kubectl create -f or kubectl rolling-update`. kubectl is a veritable multi-tool of container orchestration and management. Below we describe some of the features of kubectl that you may not have seen.   
+[kubectl](https://github.com/kubernetes/website/blob/master/content/en/docs/reference/kubectl/overview.md) is the command line tool for interacting with Kubernetes clusters. Many people use it every day to deploy their container workloads into production clusters. But there’s more to kubectl than just `kubectl create -f or kubectl rolling-update`. kubectl is a veritable multi-tool of container orchestration and management. Below we describe some of the features of kubectl that you may not have seen.
```

Comment:
```suggestion
[kubectl](/docs/reference/kubectl/overview/) is the command line tool for interacting with Kubernetes clusters. Many people use it every day to deploy their container workloads into production clusters. But there’s more to kubectl than just `kubectl create -f or kubectl rolling-update`. kubectl is a veritable multi-tool of container orchestration and management. Below we describe some of the features of kubectl that you may not have seen.
```

Since this now refers to a link on the website itself, it should goto the website page instead of the github page.